lib/src/lifecycle_module.dart

```diff
@@ -125,11 +132,60 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
       'willUnload': willUnload,
       'didUnload': didUnload,
     }.forEach(_logLifecycleEvents);
+
+    _name = 'LifecycleModule($runtimeType)';
+  }
+
+  /// If this module is in a transition state, this is the Span capturing the
+  /// transition state.
+  ///
+  /// Example:
+  ///
+  /// ```dart
+  /// @overide
+  /// Future<Null> onLoad() {
+  ///   var value = 'some_value;
+  ///   ...
+  ///   activeSpan.setTag('some.tag.name', value);
+  ///   ...
+  /// }
+  /// ```
+  Span get activeSpan => _activeSpan;
+
+  /// Set internally by the parent module if this module is called by [loadChildModule]
+  SpanContext _parentContext;
+
+  /// Builds a span that conditionally applies a followsFrom reference if this module
+  /// was loaded by a parent module.
+  ///
+  /// Returns `null` if this module does not override the [name] getter or if
+  /// if no globalTracer is configured.
```

Comment:
Could we update this comment to say why? (tracing is disabled for unnamed modules since the default name gets mangled in dart2js)